react-frontend/src/login/LoginContainer.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+import Login from "login/LoginComponent"; /*to do this component is depent on login presentational component */
+import { store } from "root/redux-store";
+import UnauthorisedErrorComponent from "shared-components/UnauthorisedErrorComponent";
+import InternalServerErrorComponent from "shared-components/InternalServerErrorComponent";
+
+const LoginContainer = () => {
+  const loginAuthorization = useSelector((state) => state.loginReducer);
+
+  const responseGoogleOnSuccess = ({ tokenObj }) => {
+    store.dispatch({ type: "LOGIN_API", payload: tokenObj.access_token });
+  };
+
+  const responseGoogleOnFailure = (error) => {
+    store.dispatch({ type: "LOGIN_FAILURE", value: error });
+  };
+
+  if (loginAuthorization.error.message === "unauthorized user") {
+    return <UnauthorisedErrorComponent />;
+  } else if (loginAuthorization.error.error === "popup_closed_by_user") {
```

Comment:
why different keys? sometimes error.error and sometimes error.message?

Reply:
error.error is the error send by google and error.message is the error which will come from api

Reply:
I checked the wiki doc: https://github.com/joshsoftware/peerly/wiki/API-documentation#error-response-1. Couldn't find any reference for `error.error`.

@onkar-josh can u please help me understand from where did u get this error structure?

Reply:
actually this error is not part of API. this error occurs when we close pop-up window of google.